scripts/globals/spells/sneak.lua

```diff
@@ -1,33 +1,29 @@
 -----------------------------------------
 -- Spell: Sneak
--- Lessens chance of being detected by sound
--- Duration is random number between 30 seconds and 5 minutes
+-- Lessens chance of being detected by sound.
+-- Duration is random number between 30 seconds and 5 minutes.
 -----------------------------------------
 require("scripts/globals/magic")
 require("scripts/globals/msg")
 require("scripts/globals/settings")
 require("scripts/globals/status")
+-----------------------------------------
 
-function onMagicCastingCheck(caster,target,spell)
+function onMagicCastingCheck(caster, target, spell)
     return 0
 end
 
-function onSpellCast(caster,target,spell)
-    if (not target:hasStatusEffect(dsp.effect.SNEAK)) then
+function onSpellCast(caster, target, spell)
+    if not target:hasStatusEffect(dsp.effect.SNEAK) then
 
-        -- last 7-9 minutes
-        local duration = calculateDuration(math.random(420, 540), spell:getSkillType(), spell:getSpellGroup(), caster, target)
-        duration = calculateDurationForLvl(duration, 20, target:getMainLvl())
+        local duration = calculateDuration(math.random(42, 54) * 10, spell:getSkillType(), spell:getSpellGroup(), caster, target)
```

Comment:
This is making the duration be locked to 10 second intervals. If that's intentional, I'm not sure where the proof for that behavior exists.

Reply:
he asked me about it and I ok'd it because when do you see 93 seconds or 231 second durations ever. I figure when SE beefed up the durations on retail they just multiplied it. didn't realize what I was really okaying I was thinking of the setting. my bad.

Reply:
actually i still think it'd be ok. we won't know if its accurate or not though without a lot of checking retail with same items

Reply:
I'll dump it from this PR and if I can find support add it separately, since my thoughts on this would extend to pots and jigs and stuff also.